rust/lance/src/dataset/scanner.rs

```diff
@@ -957,17 +957,18 @@ impl Scanner {
         let plan = self.create_plan().await?;
         // Datafusion interprets COUNT(*) as COUNT(1)
         let one = Arc::new(Literal::new(ScalarValue::UInt8(Some(1))));
-        let count_expr = create_aggregate_expr(
-            &count_udaf(),
-            &[one],
-            &[lit(1)],
```

Comment:
It's not obvious to me what this is supposed to map to in the AggregateExprBuilder pattern.

Reply:
Using the builder seems fine (and more readable).  +1